config/application.rb

```diff
@@ -29,13 +29,12 @@ class Application < Rails::Application
     # config.i18n.load_path += Dir[Rails.root.join('my', 'locales', '*.{rb,yml}').to_s]
     # config.i18n.default_locale = :de
 
-    # Do not swallow errors in after_commit/after_rollback callbacks.
-    config.active_record.raise_in_transactional_callbacks = true
-
     config.log_tags = [:host, :uuid]
 
     config.cache_store = :memory_store
 
     config.active_job.queue_adapter = :delayed_job
+
+    config.action_controller.raise_on_unfiltered_parameters = true
```

Comment:
This will be the default in Rails 5.1 -- basically, in the old world, there are a bunch of Hash methods you could call on a params hash (`each_with_index`, `merge`, etc) that would implicitly ignore strong parameters. In the new world, ActionController::Parameters does not inherit from Hash, and if you cast it to a hash in any way without strong params, it'll blow up.

I also had to upgrade the airbrake gem because it was doing exactly that internally when reporting errors.

Reply:
Word of caution with upgrading airbrake. They changed their method definitions in a few places at some point.

Reply:
Specifically I think they changed the attribute name for the API key and possibly the notify_or_ignore method.